test/yulPhaser/Chromosome.cpp

```diff
@@ -47,25 +47,25 @@ using namespace solidity::util;
 namespace solidity::phaser::test
 {
 
+vector<string> const ChrOmOsoMeSteps{
```

Comment:
Why the weird capitalization?

Reply:
Oh, I see!

Reply:
Yeah. I could not come up with a better name for it :)